Services/LDAP/classes/class.ilLDAPRoleMappingTableGUI.php

```diff
@@ -25,7 +25,7 @@ class ilLDAPRoleMappingTableGUI extends ilTable2GUI
     /**
      * @throws ilCtrlException
      */
-    public function __construct($a_parent_obj, $a_server_id, $a_parent_cmd = '')// TODO PHP8-REVIEW Tpye hints are missing here
+    public function __construct(?object $a_parent_obj, int $a_server_id, string $a_parent_cmd = '')
```

Comment:
IMO you can assume that the first paremeter MUST not be `null`. You can remove the `?` and the check below,